### PR TITLE
Add validation on redemption to check for assets in trading

### PR DESCRIPTION
**Pull request by iherger**

### CODE REVIEW

File: src/components/Routes/Fund/FundInvestRedeem/FundRedeem/FundRedeem.tsx [hunk 1]
```diff
@@ -98,6 +104,19 @@ export const FundRedeem: React.FC<FundRedeemProps> = ({ address }) => {
     );
   }
 
+  if (lockedAssets?.length) {
+    return (
+      <Block>
+        <SectionTitle>Redeem</SectionTitle>
+        <p>Redemption is currently not possible, because the fund has assets locked in the trading contract.</p>
```
Comment (jennazenk) [inline src/components/Routes/Fund/FundInvestRedeem/FundRedeem/FundRedeem.tsx]: I would change this message to something like: "Some assets of the fund are in the trading contract. The manager needs to release the assets from the trading contract back to the vault contract to enable full redemption.

File: src/components/Routes/Fund/FundInvestRedeem/FundRedeem/FundRedeem.tsx [hunk 2]
```diff
@@ -98,6 +104,19 @@ export const FundRedeem: React.FC<FundRedeemProps> = ({ address }) => {
...
+        <RequiresFundManager fallback={false}>
+          As the fund manager, you can on <Link to={`/${prefix}/fund/${address}/manage`}>move the assets to vault</Link>
```
Comment (jennazenk) [inline src/components/Routes/Fund/FundInvestRedeem/FundRedeem/FundRedeem.tsx]: if it is the fund manager redeeming, and there are assets in the vault, why not trigger the function to return assets to vault in this case?

Comment (fubhy) [reply src/components/Routes/Fund/FundInvestRedeem/FundRedeem/FundRedeem.tsx]: Chained transactions are still not easy and we have a work-in-progress issue for making them more accessible. Once that's done we can come back to this and automatically trigger it for fund managers.

Comment (jennazenk) [reply src/components/Routes/Fund/FundInvestRedeem/FundRedeem/FundRedeem.tsx]: Right, but we could do it the same way it's done for  approve/invest maybe?

**Pull request merged**